lib/log_tools/exporter.rb

```diff
@@ -150,11 +150,20 @@ def export(*logfiles)
 
                     exporter = exporter_class.new(stream)
                     index = 0
+                    start_index_local = 0
+                    if start_index
+                        start_index_local = start_index
+                    end
+                    index_increment_local = 1
```

Comment:
A more ruby-ish to do this would be

~~~
index_increment_local = index_increment || 1
~~~

**However**, at the level of this class, the value in `start_index` and `index_increment` should really always be valid (no `nil`s) and initialized to a sane default.